File: 1301-1400/1313-Decompress Run-Length Encoded List/1313-Decompress Run-Length Encoded List.py
class Solution:
    def decompressRLElist(self, nums: List[int]) -> List[int]:
        result = []
        for i in range(0, len(nums), 2):
            a, b = nums[i], nums[i + 1]
            result += [b] * a
        return result

File: 1301-1400/1313-Decompress Run-Length Encoded List/1313-Decompress Run-Length Encoded List.cpp
class Solution {
public:
    vector<int> decompressRLElist(vector<int>& nums) {
        vector<int> result;
        for (int i = 0; i < nums.size(); i += 2) {
            for (int j = 0; j < nums[i]; ++j) {
                result.push_back(nums[i + 1]);
            }
        }
        
        return result;
    }
};
